cocodump/reader.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+
+def read_img(file: Path) -> bytearray:
+    lines = file.read_text().split('\n')[1:]
+
+    image: bytearray = bytearray()
+
+    for line in lines:
+
+        if line.startswith('#') or len(line) == 0:
+            pass
+        elif '*' in line:
+            (amount, byte) = line.split('*')
+            amount = int(amount)
+            byte = int(byte, 16)
+
+            for _ in range(amount):
+                image.append(byte)
+        else:
+            image.append(int(line, 16))
+
+    return image
```

Comment:
I guess, in this case it will be better to use `with open(file) as file_obj:` instead of `Path.read_text()`. You just to need call `iter` on `file_obj` and call `next` once, then just pass the created iterator to `for` loop.

Reply:
This function is temporary and will be reimplemented as part of the library.